interface-definitions/include/auth-local-users.xml.i

```diff
@@ -7,6 +7,10 @@
     <tagNode name="username">
       <properties>
         <help>Username used for authentication</help>
+        <valueHelp>
+          <format>name</format>
```

Comment:
This should be `txt` as it will be automatically converted and placed in <> by the bash completion helpers

Reply:
Fixed